camera/src/definitions.ts

```diff
@@ -269,15 +270,15 @@ export interface GalleryPhoto {
    * The format of the image, ex: jpeg, png, gif.
    *
    * iOS and Android only support jpeg.
-   * Web supports jpeg, png and gif.
+   * Web supports png and gif.
    *
    * @since 1.2.0
    */
   format: string;
 }
 export interface GalleryImageOptions {
   /**
-   * The quality of image to return as JPEG, from 0-100
+   * The quality of image to return, from 0-100
```

Comment:
quality is only supported on android and ios, where it's always jpeg, so no need to delete it, but maybe add a note about it not being supported on web.